packages/ui/components/ui/tx/view/viewbox.tsx

```diff
@@ -17,7 +17,7 @@ export const ViewBox = ({ label, visibleContent, isOpaque }: ViewBoxProps) => {
   // if isOpaque is undefined, set it to !visibleContent
   isOpaque = isOpaque ?? !visibleContent;
   return (
-    <Box>
+    <Box className='overflow-hidden'>
```

Comment:
This is tricky, but we need a way to display the full validator address here (without the window extending beyond the view of the screen). It will be important that users can validate this on this screen. Perhaps it's something that can be clicked to expand?

<img width="348" alt="Screenshot 2024-06-04 at 1 19 37 PM" src="https://github.com/penumbra-zone/web/assets/16624263/4cb23750-8a65-44f0-a9c3-8751596d715c">

Reply:
@grod220 @Valentine1898 Thanks for the comments. I added a picture of how it was before to the description of this PR. This time, I removed the `truncate` prop from the ActionDetails.Row component and created the ActionDetails.TruncatedText component that renders long text accessibly with keyboard navigation support.


https://github.com/penumbra-zone/web/assets/29180358/79cbaa59-67d1-46ce-8237-35c1ecc68851